src/shared/clients/ecsClient.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ECS } from 'aws-sdk'
+
+export interface EcsClient {
+    readonly regionCode: string
+
+    listClusters(): AsyncIterableIterator<ECS.String>
```

Comment:
Considering these aren't structures, could we return `string` instead of `ECS.String` for these methods?

Reply:
Looks like it'll work. Typescript is full of surprises.